treeherder/log_parser/utils.py

```diff
@@ -137,7 +145,14 @@ def get_mozharness_substring(line):
     return mozharness_pattern.sub('', line).strip()
 
 
+def is_parsed(job_log_url):
+    # if parse_status is not available, consider it pending
+    parse_status = job_log_url.get("parse_status", "pending")
+    return parse_status == "parsed"
+
+
 def extract_log_artifacts(log_url, job_guid, check_errors):
+    """Generate a summary artifact for the raw (unstructured) log."""
```

Comment:
Should we use 'text' instead of 'unstructured'?
Just wondering if we should use text vs json when describing them rather than unstructured vs structured, given discussions in the past about "structured logs" being quite a nebulous term. (Yey naming is hard hehe).